projects/angular-l10n/src/lib/services/l10n-intl.service.ts

```diff
@@ -28,13 +28,15 @@ import { L10nTranslationService } from './l10n-translation.service';
      * Formats a date.
      * @param value A date, a number (milliseconds since UTC epoch) or an ISO 8601 string
      * @param options A L10n or Intl DateTimeFormatOptions object
+     * @param useDateLanguage A boolean, should a 'dateLanguage' be used or not
      * @param language The current language
      * @param timeZone The current time zone
      */
     public formatDate(
         value: any,
         options?: L10nDateTimeFormatOptions,
-        language = this.locale.language,
+        useDateLanguage = true,
+        language = useDateLanguage ? this.locale.dateLanguage || this.locale.language : this.locale.language,
```

Comment:
Ah, yes, you are right. I'll change it.
Would you try to fallback to `dateLanguage` first and than to `language`?

Reply:
We should leave it as it was:
```
    public formatDate(
        value: any,
        options?: L10nDateTimeFormatOptions,
        language = this.locale.language,
        timeZone = this.locale.timeZone
    )
```

Reply:
@dzena Maybe you were right here, it is preferable to fallback to `dateLanguage `first and than to `language`:

```
    public formatDate(
        value: any,
        options?: L10nDateTimeFormatOptions,
        language = this.locale.dateLanguage || this.locale.language,
        timeZone = this.locale.timeZone
    )
```

So if `dateLanguage `has a value, it will be used by default, even without evaluating the input parameter.

Is this a common scenario for you?

Reply:
@robisim74 
yes, i think it should be like that. So, if i've set the `dateLanguage` I would expect that it is used by default (no need to pass a `language`). 
I see that you did it already. Thanks a lot!
Cheers